addons/pos_epson_printer_restaurant/models/pos_restaurant.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+class RestaurantPrinter(models.Model):
+
+    _inherit = 'restaurant.printer'
+
+    printer_type = fields.Selection(selection_add=[('epson_epos', 'Use an Epson printer')])
+    epson_printer_ip = fields.Char(string='Epson Receipt Printer IP Address', help="Local IP address of an Epson receipt printer.")
```

Comment:
do we really need the field 'epson_printer_ip', can't we use 'proxy_ip' as they are both ip addresses, and we differciate the printers with 'printer_type'.

We'll probably have another field in future when we'll be able to select an iot device

Reply:
There's already a field to select an IoT device in Enterprise. Since the field for the IoT device hides the `proxy_ip` field, I would have to create a new module `pos_epson_printer_iot` just to override the condition to hide `proxy_ip`.

I thought it was clearer this way.